mono/mini/debugger-agent.c

```diff
@@ -4987,6 +4989,77 @@ ss_update (SingleStepReq *req, MonoJitInfo *ji, SeqPoint *sp, DebuggerTlsData *t
 	return hit;
 }
 
+#if IL2CPP_MONO_DEBUGGER
```

Comment:
Should this be an `#ifdef` of `#if defined(...)` ?

Reply:
We are evidently sticking with #ifdef because that is the Mono convention, so I need to fix this.  There is a Trello board task for cleaning up the preprocessor stuff.  I can clean up the #if/#ifdef in a future PR.